packages/presentation/src/components/ObjectSearchPopup.svelte

```diff
@@ -153,81 +154,58 @@
 
 <FocusHandler {manager} />
 
-<form
-  class="antiCard dialog completion objectPopup"
-  on:keydown={onKeyDown}
-  use:resizeObserver={() => dispatch('changeSize')}
->
-  <div class="header-dialog">
+<form class="antiPopup" on:keydown={onKeyDown} use:resizeObserver={() => dispatch('changeSize')}>
+  <div class="ap-menuHeader">
     {#if label}
-      <div class="fs-title flex-grow mb-4">
+      <div class="ap-subheader caption">
         <Label {label} />
       </div>
     {/if}
-    <div class="flex-row-center gap-1">
+    <div class="tabs">
       {#each categories as c, i}
         {@const status = categoryStatus[c._id] ?? 0}
-        <div class="ap-categoryItem">
-          <Button
-            focusIndex={i + 1}
-            kind={'transparent'}
-            showTooltip={{ label: c.label }}
-            selected={category?._id === c._id}
-            icon={c.icon}
-            size={'x-large'}
-            disabled={status === 0}
-            on:click={() => {
-              category = c
-            }}
-          />
+        <!-- svelte-ignore a11y-click-events-have-key-events -->
+        <!-- svelte-ignore a11y-no-noninteractive-tabindex -->
+        <div
+          class="tab"
+          class:selected={category?._id === c._id}
+          class:disabled={status === 0}
```

Comment:
Zero here is like magical number.
Better make function predicate to hide logic there.
And use it here like:
`isStatusDisabled(status)`